content/02.introduction.md

```diff
@@ -67,14 +72,30 @@ Coronaviruses are known to cause respiratory illnesses in humans through the fol
 Mortality also appeared to be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing.
 They also found that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
 
-
 #### Vaccines for Viruses: Strategies for and challenges to development
 
-What information is needed to develop a vaccine? How have vaccines for other viruses such as H1N1 been developed?
+Generally, the first step in producing a vaccine is characterizing the target.
+The genetic sequence of SARS-CoV-2 was released to the world on January 11, 2020, which launched an intense global effort to develop a vaccine to prevent COVID-19.
+The Coalition for Epidemic Preparedness Innovations (CEPI) is coordinating global health agencies and vaccine developers to develop vaccines against SARS-CoV-2.
+As of April 8, 2020, there are 115 vaccine candidates to prevent COVID-19, of which 78 were active. 
+Of the 78 active vaccine programs, 73 were categorized as in the preclinical or exploratory stage (@doi:10.1038/d41573-020-00073-5).
+
+Historically, an H1N1 influenza vaccine was developed relatively efficiently, mainly because influenza-vaccine technology had already been developed and regulatory agencies had already decided that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change.
+Critiques of the experience producing and distributing the H1N1 vaccine have stressed the need for alternative development-and-manufacturing platforms that can be readily adapted to new pathogens.
+Although a monovalent H1N1 vaccine was not available before the pandemic peaked in the United States and Europe, it was available soon afterward as a stand-alone vaccine that was eventually incorporated into the commercially available seasonal influenza vaccines (@doi:10.1056/NEJMp2005630).
+If H1N1 vaccine development provides any indication, considering developing and manufacturing platforms for promising COVID-19 vaccine trials early could hasten the emergence of an effective prophylactic vaccines against SARS-CoV-2. 
+
+Unlike many global vaccine development programs previously, such as with H1N1, the vaccine development landscape for COVID-19 includes vaccines produced by a wide array of technologies. 
+Experience in the field of oncology is encouraging COVID-19 vaccine developers to use next-generation approaches in vaccine development, which has led to the great diversity of vaccine development programs (@https://www.the-scientist.com/news-opinion/newer-vaccine-technologies-deployed-to-develop-covid-19-shot-67152). 
+Diverse technology platforms include DNA, RNA, virus-like particle, recombinant protein, both replicating and non-replicating viral vectors, live attenuated virus as well as inactivated virus approaches. 
```

Comment:
```suggestion
Diverse technology platforms include DNA, RNA, virus-like particle, recombinant protein, both replicating and non-replicating viral vectors, live attenuated virus, and inactivated virus approaches. 
```
I don't know much about vaccines, but I'm guessing these fall into different categories? e.g., could you have an RNA vaccine that was also an inactivated virus approach? If so, would it be possible to sign post a little more to say what each grouping refers to?

Reply:
That's a good point! I was listing as many subtypes of vaccine development as I could think of since these would be described in more detail in the Vaccines section of the paper. Perhaps:

"Diverse technology platforms include a wide variety of DNA, RNA, viral particle and recombinant protein approaches."

Maybe we can wait to provide details about replicating/non-replicating and live/inactivated later under each subsection (DNA, RNA, etc) in the Vaccines section later in the paper to avoid confusion?

Reply:
I would change the order and live-attenuated and inactivated in the beginning since these are the oldest types and then follow up with more modern approaches